WEB_SERVICE/WEB/_MEDIUM/HTTP_METHOD_MANIPULATION/description.md

```diff
@@ -0,0 +1,23 @@
+HTTP method manipulation involves exploiting inconsistencies in how GraphQL servers handle HTTP methods. In GraphQL, mutations are typically executed using the POST method to prevent sensitive operations from being exposed in the URL. However, if the server incorrectly allows mutations to be executed using GET requests, sensitive data could be exposed in URLs, leading to security vulnerabilities.
+
+If a proxy is used to route requests, the risk increases, as proxies may log these URLs, inadvertently storing sensitive information, such as API keys or user data, which could later be compromised if logs are accessed.
+
+The security implications of HTTP method manipulation in GraphQL include:
+
+- **Sensitive Data Exposure**: When sensitive information (e.g., mutation parameters) is included in a URL, it may be exposed to logs or other unintended parties.
+- **Proxy Risks**: If a proxy logs the URLs of requests, sensitive data embedded in GET requests may be stored and accessed later by unauthorized individuals.
+- **Improper Access Control**: Allowing mutations via GET requests might lead to insecure operations being performed without proper safeguards.
+
+To check if a GraphQL API is vulnerable to this, you can attempt to execute a mutation using a GET request:
+
+=== "Python"
+```python
+import requests
+
+response = requests.get("https://your-graphql-endpoint.com/graphql", 
+    params={
+        'query': 'mutation { MutationName(input: { yourField: "value" }) { resultField } }'
+    })
+```
+
```

Comment:
Something is Wrong, Try creating a ticket and try the preview there!
![image](https://github.com/user-attachments/assets/7dcbe912-1dfa-4deb-9025-0cc47ef823c2)

Reply:
fixed